File: snap_crescent_mobile-flutter/lib/models/app_config.dart
class AppConfig  {
  
  String? configkey;
  String? configValue;

  AppConfig  (
      {this.configkey,
      this.configValue,
      });


  factory AppConfig.fromMap(Map<String, dynamic> map) {
      return AppConfig(
        configkey : map['CONFIG_KEY'],
        configValue : map['CONFIG_VALUE']
      );

   }

  Map<String,dynamic> toMap() {
    Map<String,dynamic> map =  new Map();

    map['CONFIG_KEY'] = configkey;
    map['CONFIG_VALUE'] = configValue;

    return map;
  }
}

File: snapcrescent/src/main/java/com/snapcrescent/appConfig/AppConfig.java
package com.snapcrescent.appConfig;

import java.io.Serializable;

import jakarta.persistence.Entity;
import jakarta.persistence.Id;

import lombok.Data;

@Entity
@Data
public class AppConfig implements Serializable {

	private static final long serialVersionUID = -6497139748806438592L;

	@Id
	private long id;
	
	private String configKey;
	private String configValue;

}
